src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -191,6 +191,7 @@ describe("Shipping", () => {
         __typename: "Ship",
         name: "Dr Collector",
       }
+      commitMutation.mockReset()
```

Comment:
We don't need this?

Reply:
At the moment we do. I believe this is due to calling other `mockReset()`s in beforeEach blocks rather than after, so some old state is lingering from a previous test case. If you'd like to look at it together maybe we could clean it up!